### PR TITLE
WIP Enable make verify

### PR DESCRIPTION
**Pull request by marun**
Fix ``make verify`` to allow the pre-submit job added by https://github.com/kubernetes/test-infra/pull/5369 to pass.

/cc irfanurrehman shashidharatd

### CODE REVIEW

**Comment by marun** [comment]
https://github.com/kubernetes/test-infra/pull/5369 should merge before this.

File: hack/update-bazel.sh [hunk 1]
```diff
@@ -53,7 +53,7 @@ gazelle fix \
     -mode=fix
 
 # Ignore unneeded rebuild for vendored openapi
-kazel | grep -vq 'vendor/k8s.io/kubernetes/pkg/generated/openapi/BUILD' && true
+kazel 3>&1 1>&2 2>&3 3>&- | grep -vq 'vendor/k8s.io/kubernetes/pkg/generated/openapi/BUILD' && true
```
Comment (shashidharatd) [inline hack/update-bazel.sh]: some magic here. looks like cyclic redirection. Is it intentional?

File: apis/openapi-spec/swagger.json [hunk 2]
```diff
@@ -2,7 +2,7 @@
   "swagger": "2.0",
   "info": {
    "title": "Generic API Server",
-   "version": "v1.9.0"
+   "version": "v0.0.0"
```
Comment (shashidharatd) [inline apis/openapi-spec/swagger.json]: i thought we are continuing to follow the versioning of k8s releases. Are we starting federation releases from v0.0.0?

File: hack/update-bazel.sh (hunk 1 above)
Comment (marun) [reply hack/update-bazel.sh]: Yes - this is necessary to allow filtering of stderr.

**Comment by shashidharatd** [comment]
/lgtm
Thanks @marun

**Comment by marun** [comment]
@irfanurrehman Feel free to merge this or your e2e PR.  Whichever one goes first will require the other to rebase.

**Comment by irfanurrehman** [comment]
thanks @marun.
Merging this also manually as https://github.com/kubernetes/federation/pull/114 and this does not conflict.

**Pull request merged**